windows/RNSVG/PathView.cpp

```diff
@@ -0,0 +1,295 @@
+#include "pch.h"
+#include "PathView.h"
+#if __has_include("PathView.g.cpp")
+#include "PathView.g.cpp"
+#endif
+
+#include <winrt/Microsoft.Graphics.Canvas.Svg.h>
+
+#include "JSValueXaml.h"
+#include "Utils.h"
+
+using namespace winrt;
+using namespace Microsoft::Graphics::Canvas;
+using namespace Microsoft::ReactNative;
+
+namespace winrt::RNSVG::implementation {
+void PathView::UpdateProperties(IJSValueReader const &reader, bool forceUpdate, bool invalidate) {
+  const JSValueObject &propertyMap{JSValue::ReadObjectFrom(reader)};
+
+  for (auto const &pair : propertyMap) {
+    auto const &propertyName{pair.first};
+    auto const &propertyValue{pair.second};
+
+    if (propertyName == "d") {
+      m_commands.clear();
+      m_segmentData.clear();
+
+      if (propertyValue.IsNull()) {
+        m_d.clear();
+      } else {
+        m_d = propertyValue.AsString();
+        ParsePath();
```

Comment:
@mcota I think we don't need any of the parsing logic and instead can use `XamlBindingHelper.ConvertValue`(https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.markup.xamlbindinghelper.convertvalue?view=winrt-20348)